switchboard/cli/README.md

```diff
@@ -11,36 +11,40 @@ Switchboard CLI is a command-line interface tool for interacting with the Switch
   - Cancel jobs
 - Configurable via command-line arguments or config file
 
+## Installation
+
+The CLI tool is now named `tml`. Ensure it's in your system path or reference it directly using `./tml`.
+
 ## Usage
 
 ```
-switchboard-cli [OPTIONS] <SUBCOMMAND>
+./tml [OPTIONS] <SUBCOMMAND>
 ```
 
 ### Options
 
 - `-c, --config <FILE>`: Sets a custom config file
-- `-u, --api-url <URL>`: Sets the API URL directly
-- `--log`: Enable logging
+- `--log`: Enable detailed logging (Debug level)
 
 ### Subcommands
 
 1. Login
 
    ```
-   switchboard-cli login <USERNAME> <PASSWORD>
+   ./tml login <USERNAME> <PASSWORD>
    ```
 
 2. Job Management
 
    - Enqueue a job:
 
      ```
-     switchboard-cli job enqueue <SUPERVISOR_ID> <IMAGE_ID> [OPTIONS]
+     ./tml job enqueue <IMAGE_ID> [OPTIONS]
      ```
 
      Options for job enqueue:
 
+     - `--request-id <REQUEST_ID>`: Request ID (UUID)
```

Comment:
I don't know whether there's ever a good reason to specify this manually. If this is the `request_id` that's ultimately passed down from the switchboard to the supervisor, then we should change the switchboard to generate it instead. It's really only supposed to correlate requests and responses on the asynchronous websocket channel between switchboard and supervisor.

Same goes for the rendezvous servers, those will be passed by the switchboard directly in the future and are not provided by the user.

We can merge this now and change it later, both in the CLI tool and switchboard.